openy_gated_content.module

```diff
@@ -57,6 +57,7 @@ function openy_gated_content_preprocess_paragraph(&$variables) {
   }
   $plugin_instance = $identityProviderManager->createInstance($active_provider);
   $app_config = \Drupal::config('openy_gated_content.settings')->getOriginal();
+  $variables['#attached']['drupalSettings']['openy_gated_content']['logoutUrl'] = !empty($app_config['virtual_y_logout_url']) ? $app_config['virtual_y_logout_url'] : base_path() . 'user/logout';
```

Comment:
No need to add this to `drupalSettings`, config already exists in `$variables['app_settings']`.

Example how to get this setting in component:

```js
  computed: {
    ...mapGetters([
      'getAppSettings',
    ]),
  },

```

or

```js
    logoutUrl() {
      return this.$store.getters.getAppSettings.virtual_y_logout_url;
    },
```

or use `SettingsMixin` that already includes second example